### DISCUSSION TITLE
Tailwind v4 compatibility

### DISCUSSION THREAD

**Discussion by Dev-WBAP** [livewire/flux#804] [Feature Requests] [answered]
I know this is still early given that Tailwind v4.0 is only in Beta at the moment.

What are your plans for upgrading Flux to support Tailwind v4?

Did try the other day to see if it works as it is with no changes and unless I was doing something wrong it entirely broke flux

**Comment by jeffchown** [comment, marked as answer]
@Dev-WBAP See https://github.com/livewire/flux/issues/754

**Comment by joshhanley** [comment]
@Dev-WBAP at this stage there are known issues with Tailwind v4, see issue #783 for more details. We will be adding support in due course 🙂
